framework/commerce/config.js

```diff
@@ -18,7 +24,7 @@ function getProviderName() {
       ? 'shopify'
       : process.env.NEXT_PUBLIC_SWELL_STORE_ID
       ? 'swell'
-      : null)
+      : 'commercetools')
```

Comment:
I recommend changing this code to be a switch case statement to reduce too much ternary operators nested. I would also consider checking some env variable related to the commercetools config.

Reply:
I don't think I can apply the switch change since they are the base provider settings